src/Compilers/Core/Portable/CodeGen/PrivateImplementationDetails.cs

```diff
@@ -911,4 +1158,192 @@ public sealed override int GetHashCode()
             throw Roslyn.Utilities.ExceptionUtilities.Unreachable();
         }
     }
+
+    /// <summary>
+    /// A helper method used in static constructor of <see cref="DataSectionStringType"/>.
```

Comment:
>A helper method used in static constructor of <see cref="DataSectionStringType"/>. [](http://example.com/codeflow?start=8&length=82)

I think it would be good to include information what signature this method is supposed to have.